libshaderc/include/shaderc.h

```diff
@@ -113,6 +113,13 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set the compiler to do only preprocessing. The byte array result in the
+// module returned by the compilation is the text of the preprocessed shader.
+// This option override all other compilation modes, such as disassembly mode
```

Comment:
"overrides" instead of "override".  I can't explain English grammar, sorry.  :-)